packages/zosfiles/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+- Enhancement: Created zos-files edit commands to edit a dataset or uss file locally [PR #1672](https://github.com/zowe/zowe-cli/pull/1672)
```

Comment:
For the z/OS Files SDK changelog, can you replace this with the items I had listed in #1729?

> - Enhancement: Added streaming capabilities to the `Download.dataSet` and `Download.ussFile` methods.
> - BugFix: Fixed `Get.USSFile` API not respecting USS file tags.

We want to mention the new edit commands in the CLI changelog, but they are not relevant for the SDK changelog - only changes that were made to the APIs that are inside the SDK.

Reply:
oops! thanks for catching this